storagenode/monitor/monitor_test.go

```diff
@@ -43,3 +47,45 @@ func TestMonitor(t *testing.T) {
 		assert.NotZero(t, nodeAssertions, "No storage node were verifed")
 	})
 }
+
+func TestVerifyReadable(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 0, StorageNodeCount: 1, UplinkCount: 0,
+		Reconfigure: testplanet.Reconfigure{
+			StorageNodeDB: func(index int, db storagenode.DB, log *zap.Logger) (storagenode.DB, error) {
+				return testblobs.NewSlowDB(log.Named("slowdb"), db), nil
+			},
+		},
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		planet.StorageNodes[0].Storage2.Monitor.VerifyDirReadableLoop.Pause()
+
+		slowNodeDB := planet.StorageNodes[0].DB.(*testblobs.SlowDB)
+		slowNodeDB.SetLatency(10 * time.Second)
+
+		start := time.Now()
+		planet.StorageNodes[0].Storage2.Monitor.VerifyDirReadableLoop.TriggerWait()
+		duration := time.Since(start)
+		require.Less(t, duration, 5*time.Second)
```

Comment:
So I noticed this passes regardless of the latency value because the `VerifyStorageDir()` method on `SlowBlobs` does call `slow.Sleep()`: https://github.com/storj/storj/blob/a4c51929902161606f52e716c76428575fa32c42/private/testblobs/slow.go#L225-L227